homework3/semi-uber/Server/api/utilits/checkforEdit.js

```diff
@@ -0,0 +1,33 @@
+const { Truck } = require("../Schemas/truck.schema");
+const { User } = require("../Schemas/user.schema");
+const { Load } = require("../Schemas/load.schema");
+const { STATUS } = require("../../data/loadData.json");
+
+const checkForEdit = async userID => {
+  const user = await User.findById(userID);
+
+  if (user.status === "Driver") {
```

Comment:
Move user status to a separate constant as well

Reply:
done